src/cascadia/TerminalSettingsEditor/CommonResources.xaml

```diff
@@ -159,4 +137,649 @@
+                                  Grid.Row="1"
+                                  Grid.Column="2"
+                                  Width="20"
+                                  Height="20"
+                                  HorizontalAlignment="Left">
+                                <Border x:Name="SwitchKnobOn"
+                                        Width="12"
+                                        Height="12"
+                                        Margin="0,0,1,0"
+                                        HorizontalAlignment="Center"
+                                        Background="{ThemeResource ToggleSwitchKnobFillOn}"
+                                        BorderBrush="{ThemeResource ToggleSwitchKnobStrokeOn}"
+                                        CornerRadius="7"
+                                        Opacity="0"
+                                        RenderTransformOrigin="0.5, 0.5">
+                                    <Border.RenderTransform>
+                                        <CompositeTransform />
+                                    </Border.RenderTransform>
+                                </Border>
+                                <Rectangle x:Name="SwitchKnobOff"
+                                           Width="12"
+                                           Height="12"
+                                           Margin="-1,0,0,0"
+                                           HorizontalAlignment="Center"
+                                           Fill="{ThemeResource ToggleSwitchKnobFillOff}"
+                                           RadiusX="7"
+                                           RadiusY="7"
+                                           RenderTransformOrigin="0.5, 0.5">
+                                    <Rectangle.RenderTransform>
+                                        <CompositeTransform />
+                                    </Rectangle.RenderTransform>
+                                </Rectangle>
+                                <Grid.RenderTransform>
+                                    <TranslateTransform x:Name="KnobTranslateTransform" />
+                                </Grid.RenderTransform>
+                            </Grid>
+                            <Thumb x:Name="SwitchThumb"
+                                   Grid.RowSpan="3"
+                                   Grid.ColumnSpan="3"
+                                   AutomationProperties.AccessibilityView="Raw">
+                                <Thumb.Template>
+                                    <ControlTemplate TargetType="Thumb">
+                                        <Rectangle Fill="Transparent" />
+                                    </ControlTemplate>
+                                </Thumb.Template>
+                            </Thumb>
+
+                        </Grid>
+
+                        <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="CommonStates">

```

Comment:
My brain has exploded at these animation declarations. My hope is that you were able to just steal these from somewhere and not have to figure all this out yourself. Sheesh!

Reply:
Yeah I got them from the xaml github (yay open source 👍 )